utils/client_helper.go

```diff
@@ -137,6 +140,16 @@ func GetClusterInfo(ctx context.Context, client runtimeclient.Client) (*ClusterI
 	if err != nil {
 		return nil, err
 	}
+
+	ingressOperatorCrt, err := GetSecretData(ctx, "router-ca", "openshift-ingress-operator", "tls.crt", client)
+	if err != nil {
+		return nil, err
+	}
+	ingressCN, err := GetCommonNameFromCertificate([]byte(ingressOperatorCrt))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
since `ingressOperatorCrt` is only used for `GetCommonNameFromCertificate` we should probably put both calls under a single function to make it clear it's not used for `GetClusterInfo` in general and that they're connected

Reply:
This makes sense. Updated. PTAL